### ISSUE TITLE
How to save multiple meshes in one mesh file?

### ISSUE THREAD

**Issue by yingzhang1109** [mikedh/trimesh#990]

**Comment by mikedh** [comment]
Yeah you can export the scene to a scene format (like GLTF) or dump the scene into a single mesh and export that:

```
In [1]: import trimesh

In [2]: meshes = [trimesh.creation.icosphere().apply_translation([i*3, 0,0]) for i in range(10)]

In [3]: scene = trimesh.Scene(meshes)

In [4]: scene
Out[4]: <trimesh.Scene(len(geometry)=10)>

In [5]: scene.export('stuff.glb')

In [6]: dump = scene.dump(concatenate=True)

In [7]: dump
Out[7]: <trimesh.Trimesh(vertices.shape=(6420, 3), faces.shape=(12800, 3))>

In [8]: dump.export('stuff.ply')

```

**Comment by yingzhang1109** [comment]
> Yeah you can export the scene to a scene format (like GLTF) or dump the scene into a single mesh and export that:
> 
> ```
> In [1]: import trimesh
> 
> In [2]: meshes = [trimesh.creation.icosphere().apply_translation([i*3, 0,0]) for i in range(10)]
> 
> In [3]: scene = trimesh.Scene(meshes)
> 
> In [4]: scene
> Out[4]: <trimesh.Scene(len(geometry)=10)>
> 
> In [5]: scene.export('stuff.glb')
> 
> In [6]: dump = scene.dump(concatenate=True)
> 
> In [7]: dump
> Out[7]: <trimesh.Trimesh(vertices.shape=(6420, 3), faces.shape=(12800, 3))>
> 
> In [8]: dump.export('stuff.ply')
> ```

Thanks a lot!
